src/wepy/resampling/resamplers/revo.py

```diff
@@ -1,18 +1,19 @@
 import multiprocessing as mulproc
 import random as rand
 import itertools as it
-
 import logging
-from eliot import start_action, log_call
 
 import numpy as np
 
 from wepy.resampling.resamplers.resampler import Resampler
 from wepy.resampling.resamplers.clone_merge  import CloneMergeResampler
 from wepy.resampling.decisions.clone_merge import MultiCloneMergeDecision
 
-class REVOResampler(CloneMergeResampler):
-    r"""Resampler implementing the REVO algorithm.
+import pickle
```

Comment:
standard library imports at the top with the rest.

Reply:
Moved standard library module imports to the top.

Reply:
Done.